flake.nix

```diff
@@ -1,9 +1,12 @@
 {
   inputs.nixpkgs.url = "nixpkgs/nixos-unstable";
-  inputs.nixpkgs-mozilla.url = "github:garbas/nixpkgs-mozilla/flake";
+  # We need a fixed version for wasm-bindgen-cli, that corresponds to
+  # the version of wasm-bindgen in Cargo.toml
+  inputs.nixpkgs-pinned.url = "nixpkgs/nixos-21.05";
```

Comment:
I might call this input rather `nixpkgs-wasm` or `nixpkgs-website` due to all inputs are actually pinned.